server.js

```diff
@@ -339,14 +352,14 @@ try {
   server.on("error", (err) => {
     if (err.code === "EADDRINUSE") {
       console.error(
-        `❌ Port ${PORT} is already in use. Please free the port or set a different PORT environment variable.`
+        `Port ${PORT} is already in use. Please free the port or set a different PORT environment variable.`
```

Comment:
Suggesting a small wording change, for consistency with https://github.com/qlik-oss/qlik-cloud-embed-snapshot/pull/7/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR21-R22:
```suggestion
        `Port ${PORT} is already in use. Please free the port or set a different PORT environment variable in your .env file or environment.`
```